client/reader/following-edit/index.jsx

```diff
@@ -34,7 +34,7 @@ import FollowingImportButton from './import-button';
 import FeedDisplayHelper from 'reader/lib/feed-display-helper';
 import SectionHeader from 'components/section-header';
 import Button from 'components/button';
-const stats = require( 'reader/stats' );
+import stats from 'reader/stats';
```

Comment:
I've changed the default imports to single module namespace binding imports for `reader/stats` and `reader/discover/helper`. I've rebased with the updated imports.

Following up on the linter enhancement: Catching these exceptions was a lot easier with [eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import) properly integrated into the project. I'll throw up a code review for integrating this into our eslint config shortly.